### PR TITLE
fix for cyrillic font encodings

### PR DESCRIPTION
**Pull request by FrankMittelbach**
# Internal housekeeping

## Status of pull request

- Ready to merge (after adding news entry)

## Checklist of required changes before merge will be approved
- [n/a] Test file(s) added
- [x] Version and date string updated in changed source files
- [x] Relevant `\changes` entries in source included
- [x] Relevant `changes.txt` updated
- [n/a] Rollback provided (if necessary)?
- [x] `ltnewsX.tex` (and/or `latexchanges.tex`) updated

### CODE REVIEW

File: required/cyrillic/README.md [hunk 1]
```diff
@@ -1,8 +1,6 @@
 The LaTeX `cyrillic` bundle
 ===========================
 
-Release 2022-06-01 patch level 1
```
Comment (josephwright) [inline required/cyrillic/README.md:4]: I need something here so I can keep CTAN happy - I'll edit on release

**Review by josephwright — APPROVED** [review]

**Comment by FrankMittelbach** [comment]
Am 07.11.23 um 20:36 schrieb Joseph Wright:
> -Release 2022-06-01 patch level 1
> 
> I need something here so I can keep CTAN happy - I'll edit on release
> 

ah, so this is auto changed? Well sorry for removing it then

as for a release I think it is fine just to go out  2024-06 or with the 
next -dev release, whatever ...

we could also move it to main, but then it would need to be 
cherry-picked over as it is branched off develop

I think through -dev and then main is actually best, then I can write a 
news entry for the change

File: base/doc/ltnews39.tex [hunk 2]
```diff
@@ -200,6 +200,19 @@ \subsection{\pkg{verbatim}: \cs{verb} showed visible spaces}
 \githubissue{1160}
 
 
+\section{Changes to files in the \pkg{cyrillic} category}
+
+\subsection{Correct definition of \cs{k}}
+
+Ages ago, the encoding specific definitions for various accent
+commands have been altered to guard altering some parameter values
```
Comment (davidcarlisle) [inline base/doc/ltnews39.tex:208]: <  have been
> were

**Review by davidcarlisle — APPROVED** [review]

File: required/cyrillic/README.md (hunk 1 above)
Comment (FrankMittelbach) [reply required/cyrillic/README.md:4]: put something back

File: base/doc/ltnews39.tex (hunk 2 above)
Comment (car222222) [reply base/doc/ltnews39.tex]: Maybe also:

... to guard against altering ...

Comment (car222222) [reply base/doc/ltnews39.tex]: Or, even better: 

... were changed to to guard against altering ...

**Comment by muzimuzhi** [comment]
adding a link to corresponding issue #1148

File: base/doc/ltnews39.tex (hunk 2 above)
Comment (FrankMittelbach) [reply base/doc/ltnews39.tex]: to to? :-)

Comment (FrankMittelbach) [reply base/doc/ltnews39.tex]: altered (or even changed) :-)

**Pull request merged**